src/VkTermColor.php

```diff
@@ -578,4 +583,29 @@ public static function get_term_color_taxonomies() {
 		return $taxonomies;
 	}
 
+	/**
+	 * Hex値を受け取りテキスト色を判定する
+	 *
+	 * @param [type] $rgb
+	 * @return void
+	 */
+	public static function get_dynamic_text_color($hex) {
```

Comment:
@mthaichi これって受け取るのは背景色ですよね（・ｗ・？
となると
```suggestion
	/**
	 * 背景色のHex値を受け取りテキスト色を判定する
	 *
	 * @param [string] $bg_rgb
	 * @return [string] 
	 */
	public static function get_dynamic_text_color( $bg_rgb ) {
```